### ISSUE TITLE
rack-2.0.3 conflicts with rack (~> 1.5) (Gem::ConflictError)

### ISSUE THREAD

**Issue by oystersauce8** [kmikael/vienna#3]
My first stab with vienna:

```
$ thin  -R config.ru start
/usr/local/lib/ruby/2.4.0/rubygems/specification.rb:2293:in `raise_if_conflicts': Unable to activate vienna-0.4.1, because rack-2.0.3 conflicts with rack (~> 1.5) (Gem::ConflictError)
```

**Comment by oystersauce8** [comment]
I worked around it by adding a Gemfile

```
$ cat Gemfile
gem 'thin'
gem 'vienna'

```

**Comment by kmikael** [comment]
Sorry for the long wait, @oystersauce8. Just released ´v0.4.2´ with updated dependencies.
